dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -91,16 +143,46 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
         {
             try
             {
-                PushNotificationChannelManager channelManager{};
-                winrt::PushNotificationChannel pushChannelReceived{ nullptr };
+                if (IsActivatorSupported(PushNotificationRegistrationOptions::PushTrigger))
+                {
+                    ChannelDetails channelInfo{};
+                    hresult hr = CreateChannelWithRemoteIdHelper(remoteId, channelInfo);
 
-                pushChannelReceived = co_await channelManager.CreatePushNotificationChannelForApplicationAsync();
+                    if (SUCCEEDED(hr))
+                    {
+                        co_return winrt::make<PushNotificationCreateChannelResult>(
+                            winrt::make<PushNotificationChannel>(channelInfo.channelUri.get(), channelInfo.channelId.get(),  channelInfo.appUserModelId.get(), channelInfo.channelExpiryTime),
+                            hr,
+                            PushNotificationChannelStatus::CompletedSuccess);
+                    }
+                    else if (hr == E_NOTIMPL)
+                    {
+                        PushNotificationChannelManager channelManager{};
+                        winrt::PushNotificationChannel pushChannelReceived{ nullptr };
```

Comment:
NIT: do you need `nullptr` or will void-constructor do the equivalent?

Reply:
This variable should just be set. It immediately gets assigned in the next line.